### DISCUSSION TITLE
Process for building commands using multiple files that don't need installation

### DISCUSSION THREAD

**Discussion by alsoeric** [ofek/pyapp#69] [Q&A]
I'm trying to build two standalone commands. vatkins.py and bletchley.py both use utils.py and are all located in the same directory. There is one external module that will be included in the package.  Everything will be placed in a virtual environment.  

As far as I can tell from the docs, I give pyapp the command name and requirements.txt file. The usual advice for starting the requirements.txt file is "pip3 freeze," but that seems to build a list of every Python file system. Is that necessary?

**Comment by ofek** [comment]
You'll need to either turn those into packages or ship a distribution that already has what you want installed.
